static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -1,69 +1,120 @@
 define([
         'lib/config',
         'lib/storage',
-        'lodash/objects/assign'
-    ], function(config,
-                storage,
-                assign) {
+        'lodash/objects/assign',
+        'lib/geolocation'
+    ], function(
+        config,
+        storage,
+        assign,
+        geolocation
+    ) {
 
-    function selectSequentiallyFrom(array) {
-        return array[storage.local.get('gu.alreadyVisited') % array.length];
-    }
-
-    var MEMBERSHIP = 'membership';
-    var CONTRIBUTIONS = 'contributions';
+    var offerings = {
+        membership: 'membership',
+        contributions: 'contributions'
+    };
 
     var baseParams = {
         minArticles: 3,
         colourStrategy: function() {
-            return 'membership-prominent ' + selectSequentiallyFrom(['yellow', 'purple', 'bright-blue', 'dark-blue']);
+            return 'membership-prominent yellow'
+        },
+        // Agreed with Jess and Jesse
+        campaignCode: 'gdnwb_copts_memco_banner',
+        // Used for tracking the new implementation
+        interactionOnMessageShow: {
+            component: 'engagement_banner',
+            value: 'default_paypal_and_paywall'
```

Comment:
we probably don't need to put paywall_and_paypal here as it is now the default. Also, if you are changing the campaign code here, you will need to change the regex in contributions frontend, which currently checks for "cont.*banner" I believef